runtime/compiler/env/j9method.cpp

```diff
@@ -2349,7 +2370,7 @@ TR_ResolvedJ9Method::TR_ResolvedJ9Method(TR_OpaqueMethodBlock * aMethod, TR_Fron
 	  // Check for user specified FastJNI override
 	  if (TR::Options::getJniAccelerator() != NULL && TR::SimpleRegex::match(TR::Options::getJniAccelerator(), signature(trMemory)))
          {
-		 _jniProperties |=
+		   _jniProperties |=
```

Comment:
The alignment was correct before. Also, there are some TABs in here instead of spaces, even on previous lines.